lstchain/reco/r0_to_dl1.py

```diff
@@ -457,14 +509,11 @@ def r0_to_dl1(
                     length = np.rad2deg(np.arctan2(dl1_container.length, foclen))
                     dl1_container.width = width.value
                     dl1_container.length = length.value
-
                     dl1_container.prefix = tel.prefix
 
+                    # extra info for the image table
                     extra_im.tel_id = telescope_id
-                    extra_im.num_trig_pix = event.r0.tel[telescope_id].num_trig_pix
-                    extra_im.trigger_time = event.r0.tel[telescope_id].trigger_time
-                    extra_im.trigger_type = event.r0.tel[telescope_id].trigger_type
-                    extra_im.trig_pix_id = event.r0.tel[telescope_id].trig_pix_id
+                    extra_im.low_gain_mask = event.lst.tel[telescope_id].evt.pixel_status >> 2 & 1
```

Comment:
I think this line broke the tests for the mc data, because mc does not have the `lst` member of `event`

Reply:
Solved:
https://github.com/cta-observatory/cta-lstchain/pull/358/commits/4de59e4a57ae62261ff45bcde67f116481ada4ba